examples/general/abs_home_path/src/lib.rs

```diff
@@ -67,6 +68,18 @@ impl<'tcx> LateLintPass<'tcx> for AbsHomePath {
     }
 
     fn check_expr(&mut self, cx: &LateContext<'tcx>, expr: &'tcx Expr) {
+        // Skip in build scripts
+        let is_build_script = cx
+            .sess()
+            .opts
+            .crate_name
+            .as_ref()
+            .map_or(false, |crate_name| crate_name == "build_script_build");
+            
+        if is_build_script {
+            return;
+        }
```

Comment:
Could the `let` and this `if` be combined so that there is no declaration and just `if cx.sess()...`?